### ISSUE TITLE
kansformers and transformer loss curve 

### ISSUE THREAD

**Issue by HaiFengZeng** [akaashdash/kansformers#2]
what's the loss looks like for the same config setting

**Comment by WhatMelonGua** [comment]
Yes, I'm also interested about the comparison. Is Kan truly perform obviously better out of the Formula derivation than MLP ?
is it worth to use a Kan replacing the MLP?
And to a img or matrix-like, how can we understand the formula Kan Learned.
